public/style.css

```diff
@@ -1,90 +1,239 @@
-  height: auto;
-  display: block;
-  margin-top: 2em;
-  margin-left: 2em;
+.welcome-page-div{
+  width: 700px;
+  height:350px;
+  background-color: var(--beige) ;
+  border: 3px solid var(--light-brown);
+  border-radius: 20px;
+  position: relative;
 }
 
-* {
-  box-sizing: border-box;
-  -webkit-tap-highlight-color: transparent;
+.welcome-page-heading{
+  font-size: 70px;
+  padding: 80px;
+  padding-bottom: 50px;
 }
 
-html,
-body {
-  height: 100%;
+.name-label {
+  font-size: 20px;
 }
 
-body {
+.name-input{
+  position: absolute;
+  left: 230px;
+  margin-top: 40px;
+  width: 250px;
+  height: 50px;
+  border: 2px solid;
+  border-radius: 10px;
+  caret-color: var(--negative-color);
+  font-family: 'Press Start 2P', cursive;
+}
+
+.name-input:focus{
+  font-family: 'Press Start 2P', cursive;
+}
+
+.toad{
+  width: 90px;
+  position: absolute;
+  top: 360px;
+  left: 410px;
+  z-index: -1000;
+}
+
+.toad1{
+  width: 90px;
+  position: absolute;
+  top: 360px;
+  left: 190px;
+  z-index: -1000;
+}
+
+/* QUESTION PAGE */
+.status-bar {
+  
   display: flex;
+  width: 68vw;
+  margin: 0 auto;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
 }
 
+.centered {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.scrollable {
+  height: auto;
+  max-height: 80vh;
+  overflow: auto;
+}
 
 .progress-container {
-  height: 0.8rem;
-  width: 32rem;
+  height: 1.4rem;
+  width: 50vw;
   border-radius: 0.4rem;
-  background: #000;
+  background: var(--white);
+  position: relative;
 }
 
 .progress-container #progress {
   height: 100%;
   width: 0;
   border-radius: 0.4rem;
-  background: #ff4754;
+  background: var(--beige);
   transition: width 0.4s ease;
+  position: relative;
 }
 
-li:hover {
-  color: rgb(32, 32, 238);
-  cursor: pointer;
-  
+/* MARIO PROGRESS */
+.progress-container #progress::after {
+  content: "";
+  display: block;
+  width: 50px;
+  height: 50px;
+  position: absolute;
+  top: -28px;
+  right: -45px;
+  background-image: url("/browser-quiz-project-starter/public/pngimg.com\ -\ mario_PNG72.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+
+.points-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+
+.points {
+  margin-bottom: 2.2rem;
+}
+
+.image-container {
+  background-image: url('/browser-quiz-project-starter/public/image\ \(1\).png');
+  background-size: cover;
+  width: 100px;
+  height: 100px;
 }
 
+.main-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  background-color: var(--beige);
+  border: 4px solid var(--light-brown);
+  border-radius: 10px;
+  margin: 0 auto;
+  height: 50vh;
+  width: 70vw;
+}
+
+
 .yes {
-  background-color: rgb(58, 111, 58);
+  background-color: var(--positive-color);
 }
 
 .no {
-  background-color: rgb(228, 56, 56)
+  background-color: var(--negative-color)
 }
 
 
+
+
```

Comment:
A few comments about how the style looks: Overall, very well cool look! I like how you are using custom properties throughout the script. I also really like the use of flexbox. There are a few style changes I would recommend be implemented: 

1. On the welcome page, the content is not centered in the middle, height-wise. I would try to center all of the content both length-wise and height wise. 
2. The start button is not centered between the two toads on the welcome page. Is this intentional?
3. Make sure that the div that contains the answer choices is always big enough for the text. I am noticing on question 3, the answer choices go outside of the parent div bounds. Implementing some logic or CSS method to prevent this from happening would make the design look cleaner. Let me know if you have questions on how to do this!
4. The 'new game' button on the last page is all the way at the bottom of the screen and off center